frontend/public/components/search.tsx

```diff
@@ -214,7 +224,7 @@ const SearchPage_: React.FC<SearchProps> = (props) => {
                   node: (
                     <>
                       <ResourceIcon kind={resourceKind} />
-                      {kindForReference(resourceKind)}
+                      {getChipText(resourceKind)}
```

Comment:
It might be easier to just inline since the logic pretty simple.

```suggestion
                      {model.labelKey ? t(model.labelKey) : model.label}
```

Reply:
We don't have the model; it's hidden in a helper function. I can try refactoring tomorrow.

Reply:
I looked into refactoring, but it would a lot of complexity to this component. This component pretty much just deals with a reference rather than the kind. (The name is not correct.) Like getToggleText, you need to convert the reference into the model to do this.